### PR TITLE
Describe additonal JVM config option

### PR DESCRIPTION
**Pull request by dvanherbergen**
Makes #109 unnecessary. 

Signed-off-by: Davy Vanherbergen <davy.vanherbergen@gmail.com>

### CODE REVIEW

File: docs/sources/getting-started.md [hunk 1]
```diff
@@ -73,6 +73,15 @@ export OPENHAB_HTTP_PORT=8011
 export OPENHAB_HTTPS_PORT=8444
 ```
 
+## Configuring additional JVM options
```
Comment (kaikreuzer) [inline docs/sources/getting-started.md]: Maybe we should directly refer to the cases, where this is needed (we are here on a "getting-started"), which is specifically defining a non-standard serial port.

Comment (dvanherbergen) [inline docs/sources/getting-started.md]: Text updated to make it more clear it is not needed by default.

**Comment by maggu2810** [comment]
Instead of using EXTRA_JAVA_OPTS or OPENHAB_JAVA_OPTS I would prefer to edit the system.properties as it is already done for other declarations:
https://github.com/openhab/openhab-distro/blob/66fc97431479c7b7c8a4ce635bf2f823da976107/distributions/distribution-resources/src/main/resources/etc/system.properties#L137

**Comment by dvanherbergen** [comment]
As far as I understood, users shouldn't edit anything in the runtime folder because it would be replaced when they install a new version of openHAB.

**Pull request merged**